frontend/pipeline.yml

```diff
@@ -202,21 +214,21 @@ build:frontend:docker:
   artifacts:
     expire_in: 2w
     paths:
-      - frontend/coverage
-      - frontend/screenshots
-      - frontend/logs
-      - frontend/junit
+      - ${COVERAGE_DIR}
+      - ${SCREENSHOTS_DIR}
+      - ${LOGS_DIR}
+      - ${JUNIT_DIR}
     reports:
       junit:
-        - frontend/junit/results.xml
+        - ${JUNIT_DIR}/results.xml
     when: always
   tags:
     - hetzner-amd-ax42
 
 test:frontend:acceptance:
   extends: .template:test:frontend:acceptance
   script:
-    - GUI_REPOSITORY=$(pwd)/frontend SERVER_ROOT=$(pwd) ./frontend/tests/e2e_tests/run
+    - GUI_REPOSITORY=${FRONTEND_DIR} SERVER_ROOT=${CI_PROJECT_DIR} ${FRONTEND_DIR}/tests/e2e_tests/run
```

Comment:
I think the script itself actually defines these variables as long as it's executed within the git repository.
```suggestion
    - tests/e2e_tests/run
```